src/es6/SirenAction.js

```diff
@@ -146,6 +146,10 @@ const _performSirenAction = function(action, fields, tokenValue) {
 				if (!entity) {
 					return window.D2L.Siren.EntityStore.remove(url.href, token);
 				}
+				const selfLink = entity.getLinkByRel('self');
```

Comment:
hmm, a little bit worrying that this wasn't already standard behaviour...

I guess it's highly unlikely? that someone would return an href with a different self link and still want it to update the store with the request url, but I'm a bit concerned that someone might have a wacky use case like that.

Probably worthwhile to advertise this change on #web-platform just in case, but I feel like it makes sense given how things like the Siren hypermedia browser works.

minor nit: could avoid the duplicate line `window.D2L.Siren.EntityStore.update` if you stored the href in a local variable.